scv-prototype-frontend/src/app/views/Private/Profile/EmailAddresses/EmailAddresses.jsx

```diff
@@ -34,10 +45,12 @@ const EmailAddresses = ({ emailAddresses }) => {
 };
 
 EmailAddresses.propTypes = {
-  emailAddresses: PropTypes.arrayOf(PropTypes.shape(EmailAddress.propTypes))
+  programs: PropTypes.arrayOf(PropTypes.object),
+  emailAddresses: PropTypes.arrayOf(PropTypes.object)
```

Comment:
It would be better to use `PropTypes.shape` here.

Reply:
Fixed in next commit.